README.md

```diff
@@ -24,12 +24,24 @@ Por el contrario, una máquina virtual (VM) ejecuta un sistema operativo "invita
 
 ## Mi primer contenedor
 ```bash
+# Para descargar una image sin ejecutarla
+docker pull <image_name>
+
 # Para lanzar mi primer el contenedor
+docker run --name my-first-container hello-world
+
+# Para lanzar mi primer el contenedor desacoplado de la terminal.
 docker run -d --name my-first-container hello-world
 
+# Para lanzar mi primer el contenedor desacoplado de la terminal .
+docker run -d --name my-first-container -p <desired_port>:<Source_port> hello-world
```

Comment:
That command could be better as a comment and we could change the variables **desired_port** and **source_port** to other name that describe themselves as **local_machine_port** and **container_port**.